maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlSwipe.kt

```diff
@@ -79,6 +77,47 @@ class YamlSwipeDeserializer : JsonDeserializer<YamlSwipe>() {
         }
     }
 
+    private fun resolveCoordinateSwipe(root: TreeNode, duration: Long): YamlSwipe {
+        when {
+            isRelativeSwipe(root) -> {
+                val start = root.path("start").toString().replace("\"", "")
+                val end = root.path("end").toString().replace("\"", "")
+                check(start.contains("%") && end.contains("%")) {
+                    "You need to provide start and end coordinates with %, Found: (${start}, ${end})"
+                }
+                val startPoints = start
+                    .replace("%", "")
+                    .split(",")
+                    .map { it.trim().toInt() }
```

Comment:
@amanjeetsingh150 Won't this crash when this isn't an integer? Also, if we are parsing the percentages here - why do we re-parse later upon executing the swipe?